### DISCUSSION TITLE
[remark-stringify] Avoiding unnecessary escaping in links

### DISCUSSION THREAD

**Discussion by Trott** [remarkjs/remark#885] [Q&A] [answered]
Let's say I have this in a file called `in.md`:

```markdown
[a](http://example.com/a_b_c.html)
```

And I run it through remark:

```
remark in.md
```

I get output like this

```markdown
[a](http://example.com/a\_b\_c.html)
```

Is it possible to configure remark so that it does not perform the escaping of the `_` characters in the link? (I want people reading the raw text to be able to copy/paste the link into browsers. Perhaps there is another way to accomplish this?) I'm not sure if the absence of escaping would make the markdown invalid, but I don't *think* it does and the markdown apps I've used don't require the escaping.

**Comment by wooorm** [comment, marked as answer]
Good idea and use case! See: https://github.com/syntax-tree/mdast-util-to-markdown/commit/7b381da5bceee3c36be079e27a2bad7b3cc412c2

**Reply by wooorm** [reply]
Patched!
